tests/compose/test-misc-tweaks.sh

```diff
@@ -183,4 +187,4 @@ if runcompose |& tee err.txt; then
     assert_not_reached "Successfully composed with add-files for /var/lib?"
 fi
 assert_file_has_content_literal err.txt "Unsupported path in add-files: /var"
-echo "ok bad add-files"
+echo "ok bad add-files"
```

Comment:
Stray newline removal = something your editor is doing?

Reply:
oops, I'll blame it on the editor... (but it could have easily been me as well, tbh)